frontend/package.json

```diff
@@ -77,6 +77,7 @@
     "eslint-plugin-react": "^7.29.4",
     "prettier": "2.5.1",
     "sass": "^1.51.0",
-    "typescript": "^4.4.4"
+    "typescript": "^4.4.4",
+    "@babel/plugin-proposal-private-property-in-object": "^7.16.7"
```

Comment:
@mhaswell-bcgov 

This was a minor tweak, but I see an opportunity to fix some of the Npm issues on the frontend.

I'm told you are focusing on Tech Debt, so pls let me know if you have any Ideas for a JIRA ticket on fixing these 19 high level and 9 critical vulnerabilities that Npm outputs at the console when we do `npm i && npm run start`:
<img width="554" alt="image" src="https://github.com/bcgov/gdx-agreements-tracker/assets/122112933/8ac6f74c-4582-4a40-b1c0-81b2c479434d">


I'd be interested in helping to solve some of the issues happening in the frontend's 'backend' if you know what I mean.

Reply:
npm audit outputs some interesting stuff that may be vital to clean up some of the dependency issues I'm seeing.

Reply:
Please write a ticket for this next time.

Reply:
I think we should be using `npm ci` instead of `npm i` (I wish npm made it clearer what they do). `npm i` will update the package.lock whereas `npm ci` will use exactly the versions of the dependencies that are listed in the lock.
So I'd only run `npm i` if I was intentionally changing/updating our packages. But yeah I think we should try to stay on top of the `npm audit` vulnerabilities. A lot of the time they're fairly easy to solve but sometimes it requires us updating major versions of our dependencies which can break our code.

Reply:
I didn't know that about `npm ci` - that's a good reason to use it, since it will probably prevent dependency problems

Though I wonder if it would be better to first get the vulnerabilities and other glitches fixed before freezing the versions of dependencies?

Reply:
The problem though is that the vulnerabilities get discovered/reported after the fact, so often we can fix all the vulnerabilities and lock the versions down and then a few months go by and now `npm audit` is showing a bunch of new vulnerabilities that weren't there when we fixed them before.

Reply:
that's a pain point for sure. I guess that means it doesn't matter if it's frozen or not.

I'll try using `npm ci` from now on unless I'm adding packages or updating them as part of a ticket that specifically requires those changes.